src/controllers/AuthController.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable camelcase */
+import db from '../models';
+import JWTToken from '../utils/jwt';
+import { randomPw } from '../utils/helpers';
+import hash from '../utils/hash';
+import config from '../config';
+
+export const getTokenAfterSignIn = async (res, user) => {
+  const userDetails = user.dataValues;
+  const token = await JWTToken.signToken(userDetails);
+  res.redirect(`http://localhost:${config.PORT}/api/v1/auth/access?token=${token}`);
+};
+
+/** AuthController Class */
+class AuthController {
+  /**
+   * @static
+   *
+   * @param {object} req - request object
+   * @param {object} res - response object
+   *
+   * @returns {object} - object containing a user object and token
+   * @memberof AuthController
+   *
+   */
+  static async facebookSignin(req, res) {
```

Comment:
We agreed to avoid using `static` methods, any reason for its use here?

Reply:
> @gildniy Kindly investigate why Travis is reporting a failed build.

Still didn't figure out why of those errors.